doc/modules/changes/20220630_haoyuan

```diff
@@ -0,0 +1,9 @@
+New: The values of the properties of material phases can be entered
+by the differences between adjacent phases. If this is the case,
+then all the phase properties need to be entered by a value for the first phase
+followed by M-1 values for the differences between adjacent phases. 
+Otherwise, the phase properties are entered by M values, one for each phase.
+Also note that when the option is true, the differences of the log values rather than
```

Comment:
What is the reason for handling the prefactors differently? This needs to be documented in every material model input property that uses the log values rather than the actual values. Is there a functional reason this *has* to be the log value, or could you use for example the CitcomS notation, where every value is a multiplier to the initial value (i.e. prefactor = base_prefactor * multiplier_for_this_phase). This would seem easier to understand (e.g. I was at first confused that a value of 1 in the input file means *change by one order of magnitude*, and 0 would mean *change nothing*).